src/Libuv/project.json

```diff
@@ -15,7 +15,7 @@
     ],
     "files": {
       "mappings": {
-        "/": "../../content/License.txt",
+        "/": "License.txt",
```

Comment:
Oh that. It's going to be a separate change. 1.10.0 build packages will have a slightly different structure and more changes is required.